shell/probe/raindrop.ps1

```diff
@@ -6,9 +6,18 @@ function Execute {
     try {
         $stdoutTempFile = New-Item -path "$dir\stdout.log" -Force
         $stderrTempFile = New-Item -path "$dir\stderr.log" -Force
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru -RedirectStandardOutput $stdoutTempFile -RedirectStandardError $stderrTempFile).ExitCode
-        $Code = if (Test-Path $File) {$R} Else {127}
-        return $Code
+        $proc = Start-Process -FilePath $File -NoNewWindow -PassThru -RedirectStandardOutput $stdoutTempFile -RedirectStandardError $stderrTempFile
+
+        $errVar = $null
+        $proc | Wait-Process -Timeout 45 -ErrorAction SilentlyContinue -ErrorVariable errVar
+
+        if ($errVar) {
+            $proc | kill
```

Comment:
curious if you tested / know if the early kill here affects the redirected stdout/err at all?

Reply:
It's on my list to test, but I haven't tested it yet.